frontend/app/work_packages/view_models/parent-relations-handler.js

```diff
@@ -28,7 +28,10 @@
 
 module.exports = function(CommonRelationsHandler, WorkPackageService, ApiHelper) {
   function ParentRelationsHandler(workPackage, parents, relationsId) {
-      var handler = new CommonRelationsHandler(workPackage, parents, relationsId);
+      var relations = parents.filter(function(parent) {
```

Comment:
@0xF013 LOL...

The current solution by @hirtie-maxim works. I just don't know if it is the right solution, because it looks like the error is made upstream (by the caller of `ParentRelationsHandler`).

If nobody knows anything I'll probably merge this after rebasing ^^